storage/src/conflict_manager.cc

```diff
@@ -441,6 +434,19 @@ void conflict_manager::_callback() {
         std::chrono::system_clock::time_point now0 =
             std::chrono::system_clock::now();
 
+        /* Are there index_data to remove? */
+        if (now0 >= time_to_deleted_index) {
+          try {
+            _check_deleted_index();
+            time_to_deleted_index += std::chrono::minutes(5);
+          } catch (std::exception const& e) {
+            log_v2::sql()->error(
+                "conflict_manager: error while checking deleted indexes: {}",
+                e.what());
+            _broken = true;
+            break;
```

Comment:
I don't know if these two lines are a good idea.

I know they were there before, but I think it could be better to remove those two lines and just return an error message.